Integrations/integration-ArcSightESM.yml

```diff
@@ -0,0 +1,418 @@
+description: ArcSight ESM SIEM
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Port
+  name: port
+  defaultvalue: "8443"
+  type: 0
+  required: true
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Import events as incidents
+  name: importEvents
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Events query viewer ID
+  name: viewerId
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Time field for query
+  name: timeField
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Import cases as incidents
+  name: importCases
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+script:
+  script: |-
+    var server = params.server.replace(/[\/]+$/, '') + ':' + params.port + '/';
+
+    var sendRequest = function(url, args, login) {
+        var res = http(
+                server + url + encodeToURLQuery(args),
+                {
+                    Method: 'GET',
+                    Headers: {'Content-Type': ['application/x-www-form-urlencoded']}
+                },
+                params.insecure,
+                params.proxy
+            );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            if (!login) {
```

Comment:
you can check that the args have auth token that means we are not at the log in and cancel that input to this function